db/controllers.js

```diff
@@ -1,6 +1,9 @@
-const { fetchCategories, fetchReviews, fetchReview } = require("./models")
+const { fetchCategories, fetchReviews, fetchReview, fetchCommentsById } = require("./models")
 const fs = require('fs')
 const endpoints = require('../endpoints.json')
+const { doesCategoryExist } = require("./seeds/utils")
+
+
```

Comment:
Can you rename your controllers to more suitable RESTful API naming conventions, Get/Fetch, Post/Insert etc